monitoring/mongodb/dashboard.py

```diff
@@ -1136,9 +1528,10 @@ def mongodb_state_timeline(title, expr, description="", mappings=None, **kwargs)
             [
                 Template(
                     dataSource=DATASOURCE,
-                    name="job",
-                    label="Job",
-                    query='label_values(mongodb_up{namespace="zenko"},job)',
+                    name="mongodb_instance_type",
```

Comment:
nit: `job=~"$mongodb_instance_type"` looks kind of weird → we may use `jobs` here, which is not the same as the template `job` param, and conveys the fact that this may contain more than one job (and should thus be matched with =~)